cfy_manager/components/usage_collector/scripts/script_utils.py

```diff
@@ -78,8 +92,16 @@ def collect_metadata(data):
     }
 
 
-def send_data(data, url):
+def send_data(data, url, interval_type):
     # for some reason. multi hierarchy dict doesn't pass well to the end point
```

Comment:
probably move the comment to be closer to the line it's commenting about

or just remove it, i have no idea what's it about

Reply:
Removed.